step.yml

```diff
@@ -0,0 +1,62 @@
+title: Deploy to TestFlight
+summary: Uploads binaries only to [TestFlight](https://itunesconnect.apple.com/).
+description: |
+  Uploads binaries only to [TestFlight](https://itunesconnect.apple.com/)
```

Comment:
Uploads binaries (.ipa / .pkg files) to [iTunes Connect](https://itunesconnect.apple.com/)